### DISCUSSION TITLE
Installation Process

### DISCUSSION THREAD

**Discussion by virmaior** [themactep/thingino-firmware#145] [Q&A]
First off, I'm really impressed by the progress y'all are making and the frequency of updates. If I understood the project a bit better and had any of the necessary skills and  time, I'd gladly contribute.

If I'm understanding the current install process (for T31 Atom v2 and wyze v3) https://thingino.com/cloner, I need to 
1. open the cameras up
2. short pins 5 and 6 on the flash chip
3. connect usb
4. release short

or am I misunderstanding something?

**Comment by gtxaspec** [comment]
if you have shell access on the device, @themactep has made a uniflasher script that can flash the firmware without opening the device.  keep in mind this is a total wipe, so make sure you have a backup if you ever want to return to the atom/wyze cloud.

otherwise, the steps you outlined are correct.

<https://github.com/themactep/thingino-firmware/blob/master/scripts/uniflasher.sh>

if you need real time help, shoot me an email or join us in telegram or discord, we will be happy to help.
